ConsoleAppHT1/ConsoleAppHT1/Program.cs

```diff
@@ -6,13 +6,39 @@
 
 namespace ConsoleAppHT1
 {
+    enum TransactionType
```

Comment:
Best practice for new types (enum is a new type) is to put them in a separate file.
For such small project is not an issue but in future consider moving of new types into new file.